program/Cargo.toml

```diff
@@ -19,7 +19,9 @@ solana-program-entrypoint = "3.0.0"
 solana-program-error = "3.0.0"
 solana-pubkey = "3.0.0"
 solana-rent = "3.0.0"
-solana-stake-interface = { version = "2", features = ["bincode", "borsh", "sysvar"] }
+solana-stake-interface = { path = "../interface", version = "2", features = ["bincode", "borsh", "sysvar"] }
+# TODO: Remove when mollusk updates to use latest interface package
+solana-stake-interface-legacy = { package = "solana-stake-interface", version = "2", features = ["bincode", "borsh", "sysvar"] }
```

Comment:
Noticed that the program did not have a local dependency on the interface crate. Feels a bit easier to do local dev by doing this. However, this did trip up a dep inconsistency with mollusk that would need a temp alias here.

Reply:
@joncinque maybe could comment on this, we used to use a local dependency but it was changed to a crates io dependency with fcd8b926 and i dont know if there was a reason or not

Reply:
Oh, in this case using a patch is definitely better than package aliasing. That way external packages depending on the interface crate will default to the local version as well. Will add.